spec/unit/foreman_smartproxy_rest_v3_spec.rb

```diff
@@ -22,6 +24,9 @@
   describe '#create' do
     it 'sends POST request' do
       provider.expects(:request).with(:post, 'api/v2/smart_proxies', {}, is_a(String)).returns(mock(:code => '201'))
+      provider.expects(:request).with(:get, 'api/v2/organizations', {:search => 'name="Default Organization"'}).returns(mock(:code => '201', :body => {:results => [{:id => 10}]}.to_json))
```

Comment:
These calls shouldn't be made by default as the routes won't be available, I'd suggest having a separate test for when org/locs are specified.